modules/tinymce/src/themes/silver/main/ts/ui/core/color/Options.ts

```diff
@@ -53,6 +44,14 @@ const register = (editor: Editor): void => {
     }
   };
 
+  const colorColsProcessor = (value: any): any => {
```

Comment:
```suggestion
  const colorColsProcessor = (value: unknown) => {
```

Reply:
Perhaps set the processor type on the `colorColsProcessor` variable 🤔 

https://github.com/tinymce/tinymce/blob/ece3bf774b9ae20910d073af5351f162c4fb52d5/modules/tinymce/src/core/main/ts/api/EditorOptions.ts#L17